src/verifact_agents/claim_detector.py

```
@@ -1,97 +1,324 @@
+        text: Input text to validate
+        min_length: Minimum allowed length (default: MIN_TEXT_LENGTH)
+        max_length: Maximum allowed length (default: MAX_TEXT_LENGTH)
+
+    Returns:
+        Validated text string
+
+    Raises:
+        ValueError: If text is invalid
+    """
+    if not text or not isinstance(text, str):
+        raise ValueError("Input text must be a non-empty string")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)